bin/check-a11y.js

```diff
@@ -15,10 +15,15 @@ const impactColor = {
 }
 
 const { stories } = JSON.parse(readFileSync('dist/stories.json', 'utf8'))
+const pkgFilter = process.argv.slice(2).map((pkgPath) => `./${pkgPath.split('/src')[0]}`) ?? []
 
-const storiesList = Object.keys(stories).reduce((acc, curr) => {
-  if ((stories[curr].importPath).startsWith('./packages/components/') && !(stories[curr].id).endsWith('--docs')) {
-    acc.push(stories[curr].id)
+const storiesList = Object.keys(stories).reduce((acc, cur) => {
+  if ((stories[cur].importPath).startsWith('./packages/components/') && !(stories[cur].id).endsWith('--docs')) {
+    if (pkgFilter.length && !pkgFilter.includes(stories[cur].importPath.split('/src')[0])) {
```

Comment:
Could you save these conditions into a different variables in order to make them more understandable?

E.g:
```js
if (isActualStoryFile) {
  if (isWhateverElse) {
    //...
  }
}
```